### DISCUSSION TITLE
can i undervolt my CPU if i have an intel?

### DISCUSSION THREAD

**Discussion by Nuke10** [seerge/g-helper#2112] [Q&A]
from what i read about g-helper its only support undervolt for AMD CPU's  but both with Armory crate and G-helper when i open the "Power& fans" window. (or manual mode in armoury crate) i see the PL1 and L2 slider. which i at least assume are being used for undervolting. 
and that left me a bit confused. can I or can I not undervolt the CPU using G-helper?

**Comment by seerge** [comment]
@Nuke10 PL1/PL2 are power limits (as it pretty much says there). It's **not undervolting.**

You can undervolt some Intel CPUs directly from Asus BIOS, and some by using [Intel's very own Tuning Utility](https://www.intel.com/content/www/us/en/download/17881/intel-extreme-tuning-utility-intel-xtu.html)

But many Intel CPUs are locked and can't be undervolted.

**Reply by Nuke10** [reply]
thanks, but that does not answer the question. i was asking if i can undervolt the CPU WITH G-HELPER?

**Reply by seerge** [reply]
For Intel - you can’t

**Reply by Nuke10** [reply]
ok then, what is the PL1 and PL2 sliders are for?

**Reply by seerge** [reply]
You clearly have troubles reading. I have answered this question in a first comment.

**Reply by Nuke10** [reply]
no, i just missed that part of the message, that can happen, now, now why wont you do us both a favor and cut the sarcasm.

now for next question. lets say i am using  something like Intel extreme utility or throttle stop to undervolt my CPU.
how wise it is to use with G-helper. 

for example to reduce the power going from the CPU to the GPU for dynamic boost?
or to set power limits?

**Reply by seerge** [reply]
You can try undervolting using other ways. 

It's likely won't affect power draw, but rather give you a bit higher clocks / scores.

**Reply by Nuke10** [reply]
I tought the whole point of undervolting is reducing power draw..

Will it have any effect in computer temps?

**Reply by seerge** [reply]
It's rather improves performance. Please check this section of explanation video 
https://youtu.be/hqe-PjuE-K8?t=794

**Comment by Noisrevid** [comment]
It's probably easier to just tell us what CPU you have in your Intel unit.     On 12th gen and 13th gen Intel, you can ONLY undervolt if you have a CPU ending in HX (these are unlocked).    Some 11th gens were undervoltable and mostly all 10th gen and lower were, however a lot of BIOS updates turned off the ability due to Plundervolt and you'll be screwed without a way to BIOS-mod and bench-flash the BIOS (not easy and not for the faint of heart).    

For example = 12700H = NOT UNDERVOLTABLE;  however something like the 12800HX IS undervoltable. 

And if you decide to get the new Intel Ultra 7 or 9 CPUs in the latest Zephyrus G16..  They are NOT undervoltable at all.    There are NO Ultra CPUs that have unlocked voltage (yet).
